src/libs/API.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-len */
```

Comment:
Can you place `disable` line just for the `next-line` where it's needed rather than disable it for the whole file?

Reply:
Sure.

Reply:
Done. Please check now.